.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 dist/
 MANIFEST
+examples/results/
```

Comment:
Please ensure your files have a newline at the end of them. This ensures that we're compliant with any unix-y tools that rely on that. (Which is still quite a lot!)